linked_list/linked_list.py

```diff
@@ -23,10 +23,28 @@ def __init__(self):
     self.tail = None
 
   def add_to_tail(self, value):
-    pass
+    new_node = Node(value)
+    if self.head is None:
+      self.head = new_node
+      self.tail = new_node
+    elif self.tail and self.head:
+      prev_tail = self.tail
+      prev_tail.set_next(value)
+    self.tail = new_node
 
   def remove_head(self):
-    pass
+    if self.head is None:
+      return None
+    elif self.head.next_node is None:
+      current_value = self.head
+      self.head = None
+      self.tail = None
+      return current_value
```

Comment:
how are you returning Current_value? Did this code pass the test? Will reach out to you after lecture and we will do this together.